### ISSUE TITLE
Big Bad Wolf Doesn't chase Riding Hood.

### ISSUE THREAD

**Issue by hitpointt** [Looking4Group/L4G_Core#2806]
Two weeks in a row now this fight is totally bugged. He turns someone into riding hood but doesn't chase her.

**Comment by AnonXS** [comment]
Might be linked to https://github.com/Looking4Group/L4G_Core/commits/master/src/scripts/scripts/zone/karazhan/bosses_opera.cpp or https://github.com/Looking4Group/L4G_Core/commit/3a78f9ce44f63dc1fe5a8533be8ca9aa8e8793ad#diff-38ae66543181a5d1025b56e206d0bdb8.

**Comment by honeyhoney** [comment]
Works fine for me (did it on the live realm earlier this week and just now locally). Marking as invalid for now. I'll leave this open to get a few more comments.

**Comment by AnonXS** [comment]
https://github.com/Looking4Group/L4G_Core/issues/2257

**Comment by Dekos3** [comment]
Got the same bug today. And it's not the first time, so the bug really exists. 
From my own experience, I never got thi bug during the first months of the server (maybe by chance), then I got it for the first time approximately when this thread was created, and from that moment on I got it most of the times I got Big Bad Wolf (but not all the time though), so I guess there is some random parameter triggering the bug. All I'm sure is that if Big Bad Wolf doesn't chase the first time, he won't chase during the whole fight. 

Not a game-breaking bug, but it removes all the fun of the fight, which is one of my favorite encounters in Karazhan ^^
